.rubocop.yml

```diff
@@ -108,3 +109,7 @@ Performance/Caller:
 
 Metrics/BlockLength:
   Max: 193
+
+Layout/SpaceBeforeBlockBraces:
+  EnforcedStyle: space
+  EnforcedStyleForEmptyBraces: space
```

Comment:
[CI failed](https://travis-ci.org/rspec/rspec-core/jobs/331193771) on `rubocop` linting in the first commit for 2 reasons.  The 1st is resolved by this change. The `EnforcedStyleForEmptyBraces` configuration was defaulting to `no_space` when this codebase uses `space`.